pages/about.vue

```diff
@@ -0,0 +1,14 @@
+<template>
```

Comment:
Don't mix html, javascript and css in .vue file. Instead use `src` attribute on `script`, `style` and `template` tag.